docs/rst/amldashboard/use_cases/use_cases.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_use_cases:
+
+#########
+Use Cases
+#########
+
+*AML Dashboard* addresses two primary use cases, catering to different audiences and purposes: creative professionals exploring sound variations in order to enhance creativity and create unique sound experiences; and computer scientists exploring and developing new applications using |aml| needing a practical tool to to asses the efficiency of the |aml| algorithm.
+
+.. _aml_sound_exploration: 
+
+Exploring Sound Variations with *AML Dashboard*
+===============================================
+
+.. warning::
+
+   The mentioned implementation is a work in progress and is not yet available in the current version of *AML Dashboard*.
```

Comment:
I will place the following warning at the beginning of the section to ensure it applies to both use cases mentioned

Reply:
Done